src/app/loadout-builder/filter/LockArmorAndPerks.tsx

```diff
@@ -83,6 +91,27 @@ export default memo(function LockArmorAndPerks({
       } catch (e) {}
     };
 
+  const chooseSubclass = async () => {
```

Comment:
Might be nice to share this function with the Loadouts drawer

Reply:
Can do